swarmer/src/main/kotlin/com/gojuno/swarmer/Args.kt

```diff
@@ -87,7 +87,16 @@ sealed class Commands {
                     description = "Path either relative or absolute to the file that will be used to redirect logcat of started emulator to. No redirection will happen if parameter is not presented.",
                     order = 7
             )
-            var redirectLogcatTo: String? = null
+            var redirectLogcatTo: String? = null,
+
+            @Parameter(
+                    names = arrayOf("--verbose"),
```

Comment:
Hmm, not sure it's good name for flag that controls emulators startup. I mean lots of tools have own `--verbose` flags, imagine you're seeing this in CI scripts:

```console
java -jar swarmer.jar start … --verbose
```

Would you think that it's `--verbose` for emulators or `--verbose` for Swarmer itself?